x-pack/legacy/plugins/apm/scripts/upload-telemetry-data.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// compile typescript on the fly
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Maybe add a description to this script about what it does

Reply:
aye, I could do it in this file, in dev_docs or in a README.md file in this folder, any preferences?